app/src/main/java/org/qosp/notes/ui/notebooks/NotebookFragment.kt

```diff
@@ -13,7 +13,7 @@ class NotebookFragment : MainFragment() {
     override val currentDestinationId: Int = R.id.fragment_notebook
     private val args: NotebookFragmentArgs by navArgs()
 
-    override val notebookId: Long?
+    override val notebookId: Long
```

Comment:
This is causing the build to fail

Reply:
Done. Carried out in f949e9c4118a72110ecfe55d7de3fbf63caaedbf.